src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestMetadata.cs

```diff
@@ -347,6 +350,11 @@ public IEnumerable<string> Validate()
                 yield return NuGetResources.IconMissingRequiredValue;
             }
 
+            if (Readme == string.Empty)
```

Comment:
```suggestion
            if (string.IsNullOrWhiteSpace(Readme))
```

Reply:
I just realized this isn't what we want. It's fine for users to not include a Readme element in their nuspec/project file entirely,  they just can't use an empty string there. If we logged an error for null Readme entries as well then that would mean designating 'Readme' as a required metadata element, and would break a lot of existing packages.